.github/workflows/ci.yaml

```diff
@@ -63,6 +63,35 @@ jobs:
 #            fi
 #          done
 #          bash -c "exit $code";
+  unit-tests:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Fetch History
+        run: git fetch --prune --unshallow
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: Cache Go Dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: ${{ runner.os }}-go-
+      - name: Run Unit Tests
+        run: make test
+      - name: Publish Unit Test Coverage
+        uses: codecov/codecov-action@v3
+        with:
+          name: codecov-umbrella
+          token: ${{ secrets.CODECOV_TOKEN }}
+          flags: unittests
+          files: ./cover.out,./yurttunnel-cover.out
```

Comment:
yurttunnel-cover.out file is used in openyurt repo and should be removed here, so modify as following:
files: ./cover.out,./yurttunnel-cover.out --> files: ./cover.out

Reply:
@rambohe-ch resolved